rca/account_management/forms.py

```diff
@@ -0,0 +1,12 @@
+from django import forms
+from django.contrib.auth.forms import UsernameField
+
+
+class StudentCreateForm(forms.Form):
+    # Personal Details
+    username = UsernameField(
+        widget=forms.TextInput(attrs={"autofocus": True}), required=True
+    )
```

Comment:
I've taken this `UsernameField` from django, I'm not entirely sure if I should be adding validation for this somewhere though

Reply:
Looking at the [django definition](https://github.com/django/django/blob/stable/2.2.x/django/contrib/auth/forms.py#L69) for this field, it's just a `CharField` with a little bit of unicode normalisation.

The corresponding field on AbstractUser is validated with [`UnicodeUsernameValidator`](https://github.com/django/django/blob/7f1b088ab4a4342a87a11496096471703994a006/django/contrib/auth/validators.py#L19) - so either we'll be OK if the validation from AbstractUser carries over to what we're doing, or we might want to also make use of that validator.

Reply:
Switching to a model form has allowed me to make use of the validation here